README.md

```diff
@@ -148,18 +148,17 @@ cache({
 
 Function used to generate cache keys.
 
-It determines how the cache key should be computed, receiving `req`, `res` and `normalizedUrl` as input.
+It determines how the cache key should be computed, receiving `req`, `res` and `normalizedPath` as input.
 
 Can be useful if you need to cache multiple variants of the same resource depending on the specifics of your application.
 
 **Example** - cache authenticated and non-authenticated requests separately:
 
 ```javascript
 cache({
-  getCacheKey: ({ req, normalizedUrl }) => {
+  getCacheKey: ({ req, normalizedPath }) => {
     const isAuthenticated = !!req.session.user;
-
-    return `${isAuthenticated}:${normalizedUrl}`;
+    return `${isAuthenticated}:${normalizedPath}`;
```

Comment:
There's a non-written rule to leave a blank lines between constants and return statements for legibility.

I did not take the time to add it as a eslint rule yet.